pyproject.toml

```diff
@@ -9,10 +9,12 @@ license = { text = "Modified BSD" }
 dependencies = [
     "ase>=3.23.0",
     "cython>=3",
-    "numpy>=1.26,<2",
+    # "monty==2024.7.12",  # TODO: restore once readline fixed
+    # "numpy>=1.26",  # TODO: remove after test
+    "numpy>=2.0.0",
```

Comment:
This pin should be removed, it's for test purpose, we're supporting NP2 not NP2-only 🤣